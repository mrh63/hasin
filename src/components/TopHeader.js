import React from 'react';


const TopHeader = () => {
    
    return (
        <div className="top_header">
        </div>
    )
}

export default TopHeader;