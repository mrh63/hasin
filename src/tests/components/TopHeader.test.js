import React from 'react';
import { shallow } from 'enzyme';
import TopHeader from '../../components/TopHeader';

test('should render TopHeader correctly', () => {
    const wrapper = shallow(<TopHeader />);
    expect(wrapper).toMatchSnapshot();
});