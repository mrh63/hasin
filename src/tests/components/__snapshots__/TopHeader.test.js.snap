// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render TopHeader correctly 1`] = `
<div
  className="top_header"
/>
`;
